test/method/testddec.py

```diff
@@ -48,9 +48,7 @@ def testmissingrequiredattributes(self):
             vol = volume.Volume((-4, -4, -4), (4, 4, 4), (0.2, 0.2, 0.2))
             delattr(self.data, missing_attribute)
             with self.assertRaises(MissingAttributeError):
-                trialBader = DDEC6(
-                    self.data, vol, os.path.dirname(os.path.realpath(__file__))
-                )
+                trialBader = DDEC6(self.data, vol, os.path.dirname(os.path.realpath(__file__)))
```

Comment:
can you run black on this file so there are no changes in the test?

Reply:
I've re-run black on both files to make sure. It seems to me that the PR before this wasn't cleansed with black. Thanks!